examples/semantic_segmentation.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>"This notebook shows" -&gt; "In this notebook, you will learn how to fine-tune a pretr..."</p><p> </p><p>Otherwise wording is same as first sentence, so sounds like you're repeating yourself.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>